deepinv/tests/test_models.py

```diff
@@ -16,6 +16,14 @@ def imsize():
     return c, h, w
 
 
+@pytest.fixture
+def imsize_1d():
```

Comment:
can we call it imsize_1c? I don't know where the d comes from.

Reply:
Yes, that's my bad, renaming it to `imsize_1_channel`!